Adyen/Analytics/AnalyticsProvider/AdyenContext.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import AdyenNetworking
+import Foundation
+
+// TODO: - Document
+public enum AnalyticsConfiguration {
+    case enabled(telemetry: Bool, conversion: Bool)
+    case disabled
+}
```

Comment:
maybe a struct would be better:

```Swift
public struct AnalyticsConfiguration {
     var telemetryEnabled: Bool
     var conversionEnabled: Bool
 }
```

if merchant chose both `telemetryEnabled` and `conversionEnabled` false, then the `disabled` enum case doesn't make sense, right?

Reply:
also maybe the `AnalyticsConfiguration` should be a subtype of `AdyenContext`?